### ISSUE TITLE
Endless lightning in modded game. lightning only stops when this mod is removed.

### ISSUE THREAD

**Issue by VioletSkies19** [fabric-community/spooky-time#195]
while playing a game with admittedly quite a few fabric mods alongside this mod i was mining underground when i stumbled upon a skeleton with a pumpkin on his head. A little confused i killed him. The instant he died lightning sounds began. Curious as to wtf just happened i mined my way to the surface to find lightning striking all over the place endlessly multiple times a second. Almost every pig had been turned into a zombie pigman, skeleton horse traps everywhere the eye can see. Lots of mobs randomly wearing pumpkins.  

could not be disabled with /weather clear, nor /gamerule doweathercycle false.

I wasn't sure what mod was causing it but this mod was the only wildcard that i didn't know what the mod did exactly, plus pumpkins on their heads seemed hallow-y to me. 

removing this mod ceased the endless lightning. I tried adding this mod back but the lightning just started again as soon as the world loaded.

**Comment by LemmaEOF** [comment]
Might be an interaction between two mods. What other mods do you have installed?

**Comment by VioletSkies19** [comment]
betterthanmending-1.1.1
shuffle-1.14.4-1.15.2-20w07a-1.1.1
survivaldebugstick-1.15.2-20w06a-1.0.0
Void-Fog 22.1+1.15.2
rubikshotbar-0.1.3
fairenchanting-1.1.2
Vanillaparts-1.1.2+1.15.1
grid-115-fabric0.4.23-15.1
bigbuckets-1.1.0
nomadbooks-02.1
linkedstorage-1.1.1
betterdroppeditems-10.1 
horse-stats-vanilla-1.0.4
cmdkeybind-0.4.2jar
notenoughcrashes-12.3+115.1
trinkets-2.4.1
PresenceFootsteps-r22+1.15.2
modmenu-1.10.1+build.30
Autofish-0.8.5-fabric-mc1.15
illuminations-0.4.0-115.1
SoulShards-fabric-115.2-12.0-15
diggusmaxmus-1.26
requiem-1.1.4
fabric-api-0.4.32+build.292-1.15
immersive-portals-0.12(for+mc+115+with+fabric)

**Comment by VioletSkies19** [comment]
those are all the mods that i was using with it

**Comment by Shnupbups** [comment]
Try turning off 'Spooky Weather' in the config. That should fix it.
It's an old 'feature' that never quite worked right from back when the mod was for Halloween, will be removed in future.
Apologies!

**Comment by VioletSkies19** [comment]
thanks :)

**Comment by VioletSkies19** [comment]
@Shnupbups are you referring to this in the config? or did i miss something
	"weather": {
		// Amount the thunder time is divided by. Set to 1 to disable
		"thunder_modifier": 80,
		// Make it so there are less clear skies, more rain and thunder
		"less_clear_skies": true

**Comment by Shnupbups** [comment]
Yes, that's what I'm referring to. Sorry, it's called something else in the code.

**Comment by FaultyPixel** [comment]
I have the same issue, just an endless flow of lightning strikes wich is at the same time terrifying and quite amazing. I will change the config as you have mentioned earlier but i just wanted to mention than i am using maybe five time more mods than the previous user who reported it but the only ones we have in common are : 
betterthanmending, linkedstorage, modmenu, SoulShards, diggusmaximus, fabric-api

Other than that, the mod looks awesome ! thanks !
